### PR TITLE
Polish benchmark

### PR DESCRIPTION
**Pull request by SuX97**
+ Change unit of memory to GigaByte
+ Change resize_video.py double quote to single quote
+ Add link to CSN in README
+ Change some 'x' to 'TODO'

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -5,8 +5,8 @@ Welcome to MMAction2's documentation!
    :maxdepth: 2
 
    getting_started.md
-   benchmark.md
    install.md
```
Comment (hellock) [inline docs/index.rst:8]: Move install.md above.

File: tools/data/resize_video.py [hunk 2]
```diff
@@ -23,23 +23,23 @@ def resize_videos(vid_item):
...
         os.makedirs(out_dir)
     result = os.popen(
-        f"ffprobe -hide_banner -loglevel error -select_streams v:0 -show_entries stream=width,height -of csv=p=0 {full_path}"  # noqa:E501
+        f'ffprobe -hide_banner -loglevel error -select_streams v:0 -show_entries stream=width,height -of csv=p=0 {full_path}'  # noqa:E501
     )
     w, h = [int(d) for d in result.readline().rstrip().split(',')]
     if w > h:
-        cmd = (f"ffmpeg -hide_banner -loglevel error -i {full_path} "
+        cmd = (f'ffmpeg -hide_banner -loglevel error -i {full_path} '
                f"-vf {'mpdecimate,' if args.remove_dup else ''}"
```
Comment (innerlee) [inline tools/data/resize_video.py:31]: `f'-vf {"mpdecimate," if args.remove_dup else ""}'`

File: tools/data/resize_video.py [hunk 3]
```diff
@@ -23,23 +23,23 @@ def resize_videos(vid_item):
...
     )
     w, h = [int(d) for d in result.readline().rstrip().split(',')]
     if w > h:
-        cmd = (f"ffmpeg -hide_banner -loglevel error -i {full_path} "
-               f"-vf {'mpdecimate,' if args.remove_dup else ''}"
-               f"scale=-2:{args.scale} "
-               f"{'-vsync vfr' if args.remove_dup else ''} "
-               f"-c:v libx264 {'-g 16' if args.fast else ''} "
-               f"-an {out_full_path} -y")
+        cmd = (f'ffmpeg -hide_banner -loglevel error -i {full_path} '
```
Comment (innerlee) [inline tools/data/resize_video.py:30]: note to future:
one pr should have a single purpose. mixing benchmark table changes with data processing script changes is not ideal. @SuX97

Comment (SuX97) [reply tools/data/resize_video.py:30]: It is solved [here](https://github.com/open-mmlab/mmaction2/pull/100)

**Pull request merged**